wiki/History_of_osu!/2018/en.md

```diff
@@ -4,22 +4,58 @@
+[peppy tweeted](https://twitter.com/ppy/status/956058435845611521) that the new t-shirt inventory (along with their new designs) has arrived. Six days later, they were up on the osu!store.
 
-[peppy tweeted](https://twitter.com/ppy/status/956058435845611521) that the new t-shirt inventory (along with their new designs) has arrived. Six days later, they were up on the osu!store. ([Dean Herbert on Twitter](https://twitter.com/ppy/status/958185268258865152))
+The ability to see star ratings in the osu!direct was backported from osu!lazer. Lastly, thanks to nekodex, the beatmap leaderboard redesign was added.
 
-The ability to see star ratings in the osu!direct was backported from osu!lazer. ([Dean Herbert on Twitter](https://twitter.com/ppy/status/958220599783866368)) Lastly, thanks to nekodex, the beatmap leaderboards were revamped. ([ppy/osu-web#2292](https://github.com/ppy/osu-web/pull/2292))
+peppy briefly opened a GitHub issue to discuss about *passive HP*.
 
-peppy opened a GitHub issue thread to discuss about *passive HP*. ([ppy/osu#1990](https://github.com/ppy/osu/issues/1990))
+Links:
 
-In osu!lazer, osu!mania scrolling speed was implemented. Thanks to cYsmix, a fresh set of default hit sounds were added. osu!catch's autoplay was fixed to be more human-like. ([dev changelog 20180128 – ppy blog](https://blog.ppy.sh/dev-changelog-20180128/))
+- [osu!next](http://osunext.tumblr.com/post/169336245023/hi-its-been-6-months-since-my-last-post-rest)
+- [osu!mania 7K World Cup 2018: Registrations now open!](https://osu.ppy.sh/home/news/2018-01-10-MWC7K-2018-registrations-open)
+- [Dean Herbert on Twitter](https://twitter.com/ppy/status/951884503122784257)
+- [Community Choice 2017](https://osu.ppy.sh/home/news/2018-01-18-community-choice-2017)
+- [dev changelog 20180120 – ppy blog](https://blog.ppy.sh/dev-changelog-20180120/)
+- [Dean Herbert on Twitter](https://twitter.com/ppy/status/958185268258865152)
+- [Dean Herbert on Twitter](https://twitter.com/ppy/status/958220599783866368)
+- [ppy/osu-web#2292](https://github.com/ppy/osu-web/pull/2292)
+- [ppy/osu#1990](https://github.com/ppy/osu/issues/1990)
+- [dev changelog 20180128 – ppy blog](https://blog.ppy.sh/dev-changelog-20180128/)
 
 ## February
+
+In osu!lazer, the infamous catmull styled sliders were implemented; thus allowing older beatmaps to be played the way there were meant to be played. The revert button in the options was redesigned. SS+ and S+ statistics were now visible. osu!catch's scoreboard now appears, allowing players to actually complete an osu!catch game. cYsmix has been in the works to create the new default hit sounds. Muting the client through action keys were implemented, however, the design is still in the works. In-game URL handling were added and will said to stay (opening chat channels or osu! links will use the client, instead of leaving to open via a web browser). Adjusting visual settings can be done in the "loading screen". Replay importing now works, again. 
```

Comment:
they were meant to be played
are now visible
action keys was implemented
is still in progress
was added 
will said?